### PR TITLE
chore(storybook-example): add example with storybook screenshot tests

### PR DESCRIPTION
**Pull request by sipayRT**

### CODE REVIEW

File: examples/storybook-autoscreenshots/testplane-tests/example.testplane.ts
Comment (KuznetsovRoman) [inline examples/storybook-autoscreenshots/testplane-tests/example.testplane.ts:1]: I think we should remove this file

File: examples/storybook-autoscreenshots/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "@storybook/blocks": "^8.3.0",
+    "@storybook/react": "^8.3.0",
+    "@storybook/react-vite": "^8.3.0",
+    "@storybook/test": "^8.3.0",
+    "@testplane/storybook": "^1.2.2",
+    "html-reporter": "^10.6.7",
+    "prop-types": "^15.8.1",
+    "storybook": "^8.3.0",
+    "testplane": "^8.20.1",
+    "ts-node": "^10.9.2"
```
Comment (KuznetsovRoman) [inline examples/storybook-autoscreenshots/package.json:29]: Lets add "@swc/core"

**Review by KuznetsovRoman — APPROVED** [review]
It would be nice if examples were in typescript.
We could also show examples of using manual tests with typescript:
https://github.com/gemini-testing/testplane-storybook?tab=readme-ov-file#advanced-usage

File: examples/storybook-autoscreenshots/testplane-tests/example.testplane.ts
Comment (sipayRT) [reply examples/storybook-autoscreenshots/testplane-tests/example.testplane.ts:1]: I decided to leave this example and indicated in the README that this is an example of a regular test

**Pull request merged**